CGPs/cgp-0143.md

```diff
@@ -0,0 +1,197 @@
+- **User-Friendly Enhancements**: Continuously improve the user interface and user experience based on feedback from the community. This ensures the dashboards are easy to navigate and provide the necessary insights at a glance.
+- **Comprehensive Support**: Provide ongoing support to ensure that the stablecoin dashboards meet the evolving needs of the Celo Foundation and the broader community. This includes working closely with key stakeholders, like Mento lead Marcus, to ensure the dashboards align with strategic objectives.
+- **Wallet to wallet tracking for all stablecoin currencies on Celo:**
+- Volume
+- Daily Transactions
+- Monthly Active Users (MAU)
+- Weekly Active Users (WAU)
+- Daily Active Users (DAU)
+
+# Why Artemis?
+- **Free and Open Access**: Artemis is the most suitable data provider, backed by an elite team of software engineers with extensive data engineering experience from Coinbase, Microsoft, and Google. Founded by Jon Ma and Anthony Yim, Artemis was created to democratize financial education and access, inspired by their experiences in traditional finance where high-quality investments were often exclusive. Artemis is committed to improving rationality, transparency, and accessibility of blockchain data, ensuring that all data will always be free to use and access at [app.artemis.xyz](https://app.artemis.xyz).
+- **Close Relationships with RWA and Stablecoin Builders**: Artemis works closely with key leaders in the Real World Assets (RWA) and Stablecoin space, such as Castle Island Ventures, BlueChip.org, and Standard & Poor’s. These relationships enable Artemis to stay at the forefront of developments in the industry, ensuring that our data are in the hands of the key opinion leaders of the space.
+- **Most Cited Institutional Data Provider in the Space**:
+Artemis Citations: https://docs.google.com/presentation/d/1Re7VbBr0Mf6nTtyO0yErE8qLOrtKqYqOjtpFC4ATYQM/edit#slide=id.g2e0f1db53f8_0_267
+- **Trusted by key Celo stakeholders already:** https://x.com/RegenRene/status/1807826195884696040 & https://x.com/MentoLabs/status/1794988382197571920
+
+# Funding Request
+Artemis offers comprehensive solutions with the Application Activity Monitor and Stablecoin Dashboard, priced based on the equivalent cost of hiring 2-3 full-time employees (FTEs) per year to build and maintain similar data infrastructure as well as what other ecosystems are paying for Artemis’ services.
+
+- **Application Activity Monitor**: $150,000 per year
+- **Stablecoin Dashboard**: $124,000 per year
+
+Our pricing model reflects the extensive expertise and ongoing support required to ensure these tools are continuously updated, reliable, and user-friendly.
+
+**Total Funding Request**: $274,000 per year
```

Comment:
From @work-0xj4an: update to be in CELO not USD using last 90d price

Reply:
@anthonyyim I saw here you asking for $274,000 don't know if they are Celo tokens or USD, please fix for a number in Celo and also label the number as Celo Tokens. 
I saw in the json file you are asking for 634,994 Celo Tokens. If this is the right number please fix it in this paragraph.

Reply:
Yep, will update--just finalizing the final proposal price with my team before making the edit. Thanks for your patience!

Reply:
@anthonyyim let me know when done